src/main/java/frc/robot/commands/SpeakerShootSequence.java

```diff
@@ -22,4 +25,13 @@ public SpeakerShootSequence(Conveyor conveyor, Shooter shooter) {
         Conveyor.shoot(conveyor),
         new InstantCommand(() -> shooter.setShooterMode(ShooterMode.IDLE)));
   }
+
+  /** runs SpeakerShootSequence but adds the layer that if the robot is interrupted, then everything is brought to the prep state */
+  public static Command interruptibleSpeakerShootSequence(Intake intake, Elevator elevator, Conveyor conveyor, Shooter shooter) {
+    return new SpeakerShootSequence(intake, elevator, conveyor, shooter)
+        .finallyDo(
+            (boolean interrupted) -> {
+              SpeakerPrepScoreSequence.interruptibleSpeakerPrepScoreSequence(intake, elevator, conveyor, shooter);
```

Comment:
(a) This will mean it will always go to prep score once we've scored, not just when interrupted
(b) If shooting was interrupted, then scheduling another command on top of whatever was doing the interrupting might not be a good idea?
(b) Constructing a command doesn't start it, you'd have to construct a command then do `.schedule()` on it

Reply:
I have similar comments for most of the changes in this PR